src/action_install.rs

```diff
@@ -228,11 +228,26 @@ pub fn check_tars_and_move(name: &str, dirs: &RuaDirs, archive_whitelist: &Index
 	});
 
 	for (file, file_name) in dir_items {
-		fs::rename(&file.path(), checked_tars_dir.join(file_name)).unwrap_or_else(|e| {
-			panic!(
-				"Failed to move {:?} (build artifact) to {:?}, {}",
-				&file, &checked_tars_dir, e,
-			)
-		});
+		let src = &file.path();
+		let dst = &checked_tars_dir.join(file_name);
+
+		fs::rename(src, dst)
+			.or_else(|err| {
+				const EXDEV: i32 = 18; //Invalid cross-device link
```

Comment:
I wonder on two things:
* is there any way to reference this number (`18`) from stdlib or some crate that we already have in imports? Your carefully written PR suggests that you probably tried to look it up, but doesn't hurt asking. If you tried finding it but it was fruitless, then it's OK, just asking. I'll take another glance at it tomorrow.

* are there other reasons where rename would fail, but copying wouldn't? E.g. should we always try to copy as fallback?

Reply:
UPD: I'll better take a fresh look tomorrow.